v4l2/v4l2_encode.cpp

```diff
@@ -58,6 +58,14 @@ bool V4l2Encoder::start()
     streamFormat.streamFormat = AVC_STREAM_FORMAT_ANNEXB;
     m_encoder->setParameters(VideoConfigTypeAVCStreamFormat, &streamFormat);
 
+    if (m_svct) {
+        VideoTemporalLayers layers;
+        layers.numLayersMinus1 = 1; // It is two layers actually
+        layers.bitRate[0] = 3000000;
```

Comment:
do you mean this?
layers.bitRate[0] = m_videoParams.rcParams.bitRate / 2;
layers.bitRate[1] = m_videoParams.rcParams.bitRate;

but if we don't set m_videoParams.rcParams.bitRate, how to set the layer bitrate？